app/views/local_transactions/_fields.html.erb

```diff
@@ -48,6 +48,43 @@
           <%= f.text_area :need_to_know, rows: 4, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
         </span>
       </div>
+
+      <% administrations = {
+        "Scotland" => :scotland_availability,
+        "Wales" => :wales_availability,
+        "Northern Ireland" => :northern_ireland_availability
+      } %>
+      <div class="devolved-administrations">
+        <% administrations.each do |title, field_name| %>
+          <%= f.fields_for field_name do |f| %>
+            <div class="form-group">
+              <div class="form-label">
+                <%= f.label :type, title %>
+              </div>
+              <div class="devolved-administration-fields">
+                <div class="form-wrapper">
+                  <%= f.radio_button :type, 'local_authority_service', disabled: @resource.locked_for_edits? %>
+                  <%= f.label :type, "Service available from local council", value: 'local_authority_service' %>
```

Comment:
The label looks a little fishy with the use of value, I imagine we might want to adjust the first argument of label.

You can wrap these labels around the element too which seems to be [an existing pattern](https://github.com/alphagov/publisher/blob/1a974da8b8ea342effc55ed5db78a906d59e5f1e/app/views/editions/_major_change_fields.html.erb#L19-L22) and will resolve the need for additional styling to tell it not to be a normal font size. E.g:


```suggestion
                  <%= f.label :type_local_authority_service, class: "control-label" do %>
                    <%= f.radio_button :type, 'local_authority_service', disabled: @resource.locked_for_edits? %>
                    Service available from local council
                  <% end %>
```

Reply:
Fixed up - except for the text field (URL) and label - these are all now label with form field embedded.